build/building/context.md

```diff
@@ -9,25 +9,43 @@ The [`docker build`](../../engine/reference/commandline/build.md) or
 commands build Docker images from a [Dockerfile](../../engine/reference/builder.md)
 and a "context".
 
-A build's context is the set of files located at the `PATH` or `URL` specified
-as the positional argument to the build command:
+The build context is the argument that you pass to the build command:
 
 ```console
 $ docker build [OPTIONS] PATH | URL | -
                          ^^^^^^^^^^^^^^
 ```
 
-The build process can refer to any of the files in the context. For example,
-your build can use a [`COPY` instruction](../../engine/reference/builder.md#copy)
-to reference a file in the context or a [`RUN --mount=type=bind` instruction](../../engine/reference/builder.md#run---mounttypebind)
-for better performance with [BuildKit](../buildkit/index.md). The build context
-is processed recursively. So, a `PATH` includes any subdirectories and the
-`URL` includes the repository and its submodules.
+## What is a build context?
 
-## `PATH` context
+You can pass any of the following things as the context for a build:
```

Comment:
`inputs` might be better?

```suggestion
You can pass any of the following inputs as the context for a build:
```

Reply:
Sure, sounds better to me - I struggled with this a little